compiler-plugin-test/src/test/resources/subjekt/IterationWithAggregate.yaml

```diff
@@ -0,0 +1,268 @@
+---
+name: "Invalid iteration of aggregate calls"
+config:
+  codePreamble: |-
+    /**
+      * This file has been auto-generated with Subjekt (https://github.com/FreshMag/subjekt).
+    */
+    import it.unibo.collektive.aggregate.api.Aggregate
+
+macros:
+  - def: loop(code)
+    values:
+      - |-
+        for(i in 1..3) {
+          ${{ code }}
+        }
+      - |-
+        (1..3).forEach {
+          ${{ code }}
+        }
+  - def: alignedOn(code)
+    values:
+      - |-
+        alignedOn(0) {
+          ${{ code }}
+        }
+  - def: nestedFun(code)
+    values:
+      - |-
+        fun Aggregate<Int>.nested() {
+          ${{ code }}
+        }
+  - def: aggregate(code)
+    values:
+      - |-
+        aggregate(0) {
+          ${{ code }}
+        }
+
+parameters:
+  - name: "AGGREGATE"
+    values: [ "neighboring(0)" ]
+
+# SUBJECTS
+subjects:
+
+  # SIMPLE ITERATIONS
+  - name: Iteration${{ prettify(AGGREGATE, loop(AGGREGATE)) }}
+    code: |-
+      fun Aggregate<Int>.entry() {
+        ${{ loop(AGGREGATE) }}
+      }
+    outcomes:
+      - warning: |-
+          Aggregate function '${{ AGGREGATE }}' has been called inside a loop construct without explicit alignment.
+          The same path may generate interactions more than once, leading to ambiguous alignment.
+        
+          Consider to wrap the function into the 'alignedOn' method with a unique element.
+
+  - name: |-
+      IterationExtAlign${{ prettify(AGGREGATE, loop(AGGREGATE)) }}
+    code: |-
+      fun Aggregate<Int>.entry() {
+        ${{ alignedOn(loop(AGGREGATE)) }}
+      }
+    outcomes:
+      - warning: |-
```

Comment:
Suggested improvement for Subjective: support direct elements with no list symbol (Konf does it automatically)